src/components/Planning/ProjectProgrammedDialog/ProjectProgrammedSearch.tsx

```diff
@@ -0,0 +1,169 @@
+import { getSearchResults } from '@/services/projectServices';
+import { arrayHasValue } from '@/utils/common';
+import { SearchInput } from 'hds-react/components/SearchInput';
+import { FC, memo, useCallback, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useOptions } from '@/hooks/useOptions';
+import { IProgrammedProjectSuggestions, ISearchRequest } from '@/interfaces/searchInterfaces';
+import { IClass } from '@/interfaces/classInterfaces';
+import { useAppSelector } from '@/hooks/common';
+import { selectAllClasses } from '@/reducers/classSlice';
+import SelectedProjectCard from './SelectedProjectCard';
+import usePlanningRows from '@/hooks/usePlanningRows';
+import { selectDistricts } from '@/reducers/locationSlice';
+import { ILocation } from '@/interfaces/locationInterfaces';
+
+interface ISearchState {
+  searchWord: string;
+  searchedProjects: Array<IProgrammedProjectSuggestions>;
+}
+interface IProjectSearchProps {
+  onProjectClick: (value: IProgrammedProjectSuggestions | undefined) => void;
+  projectsForSubmit: Array<IProgrammedProjectSuggestions>;
+  onProjectSelectionDelete: (projectName: string) => void;
+}
+
+const buildBreadCrumbs = (
+  path: string,
+  classes: Array<IClass>,
+  districts: Array<ILocation>,
+): Array<string> =>
+  path
+    .split('/')
+    .map(
+      (p) => classes.find((c) => c.id === p)?.name ?? districts.find((d) => d.id === p)?.name ?? '',
+    );
+const ProjectProgrammedSearch: FC<IProjectSearchProps> = ({
+  onProjectClick,
+  projectsForSubmit,
+  onProjectSelectionDelete,
+}) => {
+  const classes = useAppSelector(selectAllClasses);
+  const districts = useAppSelector(selectDistricts);
+
+  const [searchState, setSearchState] = useState<ISearchState>({
+    searchWord: '',
+    searchedProjects: [],
+  });
+  const { searchedProjects, searchWord } = searchState;
+  const phaseProposal =
+    useOptions('phases', true).find((phase) => phase.label === 'proposal')?.value || '';
+  const phaseDesign =
+    useOptions('phases', true).find((phase) => phase.label === 'design')?.value || '';
+
+  // selectedClass and selectedSubClass refer to the class and/or subclass currently selected by the user
+  // in planning view
+  // any one of these classes have to be selected by the user to be able to toggle the form
+  const { selections } = usePlanningRows();
+
+  const buildQueryParamString = useCallback(
+    (projectName: string): ISearchRequest => {
+      const reqParamObject = { limit: '30', params: '', order: 'new' };
+      const searchParams = [];
```

Comment:
Initialize the searchParams constant with values as following:

const searchParams = [`projectName=${projectName}`, `programmed=false`, `phase=${phaseProposal}`, `phase=${phaseDesign}`] and remove lines from 63-66.